### PR TITLE
Update decision apis

### PR DESCRIPTION
**Pull request by shujianchen**
As title, the three decide APIs got updated in this PR, and the version is also bumped with updated docs.

### CODE REVIEW

File: src/v1/creative-auction-bulk/types.ts [hunk 1]
```diff
@@ -24,11 +25,23 @@ export type CreativeAuctionBulkParams = {
     searchQuery?: string;
   }[];
   pageId?: string;
+  filtering?: {
+    category?: {
+      operator?: string;
```
Comment (sjhan-moloco) [inline src/v1/creative-auction-bulk/types.ts:30]:
```suggestion
      operator?: 'OR' | 'AND';
```

File: src/v1/creative-auction-bulk/types.ts [hunk 2]
```diff
@@ -50,6 +63,17 @@ export type CreativeAuctionBulkHttpRequestBody = {
     search_query?: string;
   }[];
   page_id?: string;
+  filtering?: {
+    category?: {
+      operator?: string;
```
Comment (sjhan-moloco) [inline src/v1/creative-auction-bulk/types.ts:68]: ditto

Comment (shujianchen) [reply src/v1/creative-auction-bulk/types.ts]: Resolved

**Review by sjhan-moloco — APPROVED** [review]
LGTM~

**Pull request merged**